tests/Phake/Annotation/NativeReaderTest.php

```diff
@@ -61,8 +62,11 @@ class NativeReaderTest extends TestCase
     #[\Phake\Mock(class: self::class)]
     private $mockWithNamedType;
 
-    #[\Phake\Mock(foo: self::class)]
```

Comment:
Would it be possible to keep a test where `class` is `null` ?

Reply:
I think these 2 test cases still cover this scenario where class would be `null`

https://github.com/phake/phake/blob/b33bdafae0626da64ffe0824ace43adabff7a4d1/tests/Phake/Annotation/NativeReaderTest.php#L66-L67

https://github.com/phake/phake/blob/b33bdafae0626da64ffe0824ace43adabff7a4d1/tests/Phake/Annotation/NativeReaderTest.php#L75-L76